src/registry/views/components-dependencies.pug

```diff
@@ -0,0 +1,9 @@
+#components-dependencies.box
+  each dependency in availableDependencies
+    .componentRow.row.table
+      a(href=dependency.link, target="_blank")
+        if dependency.core
```

Comment:
Core deps are subject to be whitelisted too, so I think it's useful to show them. You can see 'url' in the screenshot example